pom.xml

```diff
@@ -131,7 +131,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jitsi-xmpp-extensions</artifactId>
-      <version>1.0-44-g707c6af</version>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
This can be fixed now

Reply:
well actually not, for some reason it didn't build yet? https://github.com/jitsi/jitsi-maven-repository/commits/master

Reply:
checked with Damyan about this